src/actions/guides/database/validating-saving-deleting.cr

```diff
@@ -406,42 +399,44 @@ class Guides::Database::ValidatingSavingDeleting < GuideAction
         post = PostQuery.find(id)
         # Params contain the title and body, but not the post_id
         # So we set it ourselves
-        CommentForm.create(params, post_id: post.id) do |form, comment|
+        SaveComment.create(params, post_id: post.id) do |operation, comment|
           # Do something with the form and comment
         end
       end
     end
     ```
 
-    This sets the `post_id` when instantiating the form. You can pass anything
-    that is defined as a `column` on your model. Note that the fields are type
+    This sets the `post_id` when instantiating the operation. You can pass anything
+    that is defined as a `column` on your model. Note that the attributes are type
     safe, so you don't need to worry about typos or passing the wrong types.
     Lucky is set up to make sure it works automatically.
```

Comment:
I think we should add some notes somewhere about when to use commit v save callbacks. There are some docs in the callbacks module that mention some stuff. Like setting database columns in after_save but sending emails in after_commit